config/locales/zh.yml

```diff
@@ -1458,11 +1458,11 @@ notices.signed_up_but_unconfirmed.first_paragraph_start: 我们已发电邮到
 notices.signed_up_but_unconfirmed.resend_confirmation_email: 重新发送确认电邮
 notices.timeout_warning.partially_signed_in.continue: 继续登录
 notices.timeout_warning.partially_signed_in.live_region_message_html: '%{time_left_in_session_html} 后你会被登出。选择“保持我登录状态”保持登录；选择“把我登出”来登出。'
-notices.timeout_warning.partially_signed_in.message_html: 为了你的安全， %{time_left_in_session_html}分钟后我们将取消你的登录。
+notices.timeout_warning.partially_signed_in.message_html: 出于安全考虑， %{time_left_in_session_html}分钟后我们将取消你的登录。
 notices.timeout_warning.partially_signed_in.sign_out: 取消登录
 notices.timeout_warning.signed_in.continue: 保持我登录状态
 notices.timeout_warning.signed_in.live_region_message_html: '%{time_left_in_session_html} 后你会被登出。选择“保持我登录状态”保持登录；选择“把我登出”来登出。'
-notices.timeout_warning.signed_in.message_html: 为了你的安全，我们会在 %{time_left_in_session_html} 后将你登出，除非你告诉我们不要这样做。
+notices.timeout_warning.signed_in.message_html: 出于安全考虑，我们会在 %{time_left_in_session_html} 后将你登出，除非你告诉我们不要这样做。
```

Comment:
English: "For your security, we will sign you out in %{time_left_in_session_html} unless you tell us otherwise."

Reply:
Don't mind the box redacting my phone number; it's irrelevant. Highlighted bit of text is the change (highlighted by me to demonstrate it):

![Screenshot 2024-11-07 at 3 53 47 PM](https://github.com/user-attachments/assets/a7465d4f-3d10-4d89-9741-59db9f86ede8)